.github/workflows/continuous-integration.yml

```diff
@@ -180,6 +181,7 @@ jobs:
         php-version:
           - "8.1"
           - "8.2"
+          - "8.2"
```

Comment:
Shouldn't there be `8.3` instead of `8.2` which is already defined one line above ?